### PR TITLE
Fix output for additional extensions

### LINKED ISSUE #111: Deployment output is kind of ugly
After `Bundling additional extensions` there is the list of extensions in a weird format. There is probably some kind of call to `ls` or `echo` somewhere which prints these extensions that way.

```
 <-- Start deployment of test-php-74 -->
       Fetching source code
-----> Bundling NGINX 1.15.8
-----> Bundling PHP 7.3.12
-----> Bundling extensions
       apcu
       phpredis
       mongodb
       mcrypt
-----> Vendoring Composer
       Updating Composer
-----> Bundling additional extensions apcu
gd
imagick
lua
mongodb 
       apcu
Checksums match. Fetching from cache.
       gd
Checksums match. Fetching from cache.
       imagick
       lua
       mongodb
```

### PR DESCRIPTION
**Pull request by brandon-welsch**
Close #111 

Preview:
```bash
 <-- Start deployment of php-buildpack-test -->
       Fetching source code
       Fetching deployment cache
-----> Cloning custom buildpack: https://github.com/Scalingo/php-buildpack#fix/111/output-enhancement
-----> Bundling NGINX 1.15.8
       Checksums match. Fetching from cache.
-----> Bundling PHP 7.4.0
       Checksums match. Fetching from cache.
-----> Bundling extensions
       apcu
       Checksums match. Fetching from cache.
       phpredis
       Checksums match. Fetching from cache.
       mongodb
       Checksums match. Fetching from cache.
-----> Vendoring Composer
-----> Bundling additional extensions: curl json libxml mbstring openssl simplexml 
       curl
       Checksums match. Fetching from cache.
       json
       Checksums match. Fetching from cache.
       libxml
       Checksums match. Fetching from cache.
       mbstring
       Checksums match. Fetching from cache.
       openssl
       Checksums match. Fetching from cache.
       simplexml
       Checksums match. Fetching from cache.
Updating to version 1.9.1 (stable channel).
   Downloading (100%)
Use composer self-update --rollback to return to version 1.8.5
-----> Installing application dependencies with Composer
Loading composer repositories with package information
Installing dependencies from lock file
Nothing to install or update
Generating optimized autoload files
-----> Setting up default configuration
-----> Vendoring binaries into slug
 Build complete, shipping your container...
 Waiting for your application to boot... 
 <-- https://php-buildpack-test.osc-fr1.scalingo.io -->
```

@EtienneM It is good for you?

### CODE REVIEW

File: lib/composer [hunk 1]
```diff
@@ -56,8 +56,9 @@ function install_composer_deps() {
     cp "$target/vendor/composer/bin/composer.phar" "$target/bin/composer.phar"
 
     local required_extensions=$(jq --raw-output '.require // {} | keys | .[]' < "$BUILD_DIR/composer.json" | grep '^ext-' | sed 's/^ext-//')
+    local display_required_extensions=$(echo $required_extensions | tr '\n' ' ')
     if [ -n "$required_extensions" ]; then
-        status "Bundling additional extensions $required_extensions"
+        status "Bundling additional extensions: $display_required_extensions"
```
Comment (EtienneM) [inline lib/composer]: Actually I think we can get rid of this list of extensions as all extension name is listed below when installing it. Don't you think?
```suggestion
        status "Bundling additional extensions
```

Comment (brandon-welsch) [reply lib/composer]: Yes I agree with you

File: lib/composer [hunk 2]
```diff
@@ -68,7 +68,7 @@ function install_composer_deps() {
             elif [ "$ext" = "tidy" ] ; then
               fetch_package "libtidy-${tidy_version}" "/app/vendor/libtidy" | indent
             fi
-            fetch_package "ext/$(php_api_version)/php-${ext}" "/app/vendor/php" 2>/dev/null || true | indent
+            fetch_package "ext/$(php_api_version)/php-${ext}" "/app/vendor/php" 2>/dev/null | indent || true
```
Comment (EtienneM) [inline lib/composer]: Why this change? And actually, why all the changes except for the one on line 60 of this file?

Comment (brandon-welsch) [reply lib/composer]: This change was for testing, I go back this change, it was not necessary to update this.

For the other changes (indent() function), I wanted the functions to be the same everywhere, since the behaviour of these functions is the same. (It add principally the comments)
